read-fonts/src/collections/int_set/mod.rs

```diff
@@ -468,6 +468,27 @@ impl<T: Domain> Hash for IntSet<T> {
     }
 }
 
+impl<T: Domain + Ord> Ord for IntSet<T> {
+    fn cmp(&self, other: &Self) -> core::cmp::Ordering {
+        // there is probably room for optimization here, but it gets us working
```

Comment:
Don't need to do this now but probably worth capturing in a comment, this is what a more efficient implementation probably entails:
1. Implement Ord on BitSet, then there we can implement it by comparing page element values to each other and avoid having to produce the set members which is comparatively much slower.
2. If membership types are the same we can use the BitSet ord impl.
3. If membership types are different then use iter_ranges() which will avoid excessive slowness for exclusive sets.